generators/client/templates/react/src/main/webapp/app/modules/home/home.tsx.ejs

```diff
@@ -34,37 +34,23 @@ export interface IHomeProp {
   getSession: Function;
 }
 
-export interface IHomeState {
-  currentUser: any;
-}
-
-export class Home extends React.Component<IHomeProp, IHomeState> {
-  state: IHomeState = {
-    currentUser: this.props.account
-  };
-
-  componentWillMount() {
+export class Home extends React.Component<IHomeProp> {
+  componentDidMount() {
```

Comment:
earlire React docs stated that side effects where supposed to be done from *Will* and not from *Did*, has this changes, do you have any doc reference for this?

Reply:
@deepu 
See: [Update on Async Rendering](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html) and [Component Lifecycle Changes](https://reactjs.org/blog/2018/03/29/react-v-16-3.html#component-lifecycle-changes)